app/views/pageflow/entries/_meta_tags.html.erb

```diff
@@ -0,0 +1,4 @@
+  <meta name="dc.language" content="<%= I18n.locale %>" />
+  <% if entry.keywords.present? %><meta name="keywords" content="<%= entry.keywords %>"><% end %>
+  <% if entry.author.present? %><meta name="author" content="<%= entry.author %>"><% end %>
+  <% if entry.publisher.present? %><meta name="publisher" content="<%= entry.publisher %>"><% end %>
```

Comment:
Alternatively the render call for this template could be wrapped in a helper. Then we could test the view logic based on this helper. That's the way it works for the [SocialShareHelpers](https://github.com/codevise/pageflow/blob/master/spec/helpers/pageflow/social_share_helper_spec.rb).